uReplicator-Common/src/main/java/com/uber/stream/ureplicator/common/KafkaUReplicatorMetricsReporter.java

```diff
@@ -65,46 +66,36 @@
       _reporterMetricPrefix = null;
       return;
     }
-
-    _reporterMetricPrefix = String
-        .format(KAFKA_UREPLICATOR_METRICS_REPORTER_PREFIX_FORMAT, conf.getRegion(),
-            String.join(".", conf.getAdditionalInfo()),
-            conf.getHostname());
-    LOGGER.info("Reporter Metric Prefix is : " + _reporterMetricPrefix);
     _registry = new MetricRegistry();
-    final Boolean enabledGraphiteReporting = true;
-    final Boolean enabledJmxReporting = true;
-    final long graphiteReportFreqSec = 60L;
 
     // Init jmx reporter
-    if (enabledJmxReporting) {
+    if (conf.getEnabledJmxReport()) {
```

Comment:
getJmxReportEnabled

Reply:
for the paramter is enableJmxReport, so i modify to getEnableJmxReport